### ISSUE TITLE
convert_keras fails assertion

### ISSUE THREAD

**Issue by burgalon** [mil-tokyo/webdnn#295]
```
Traceback (most recent call last):
  File "convert_keras.py", line 62, in <module>
    main()
  File "convert_keras.py", line 37, in main
    graph = converter.convert(model, input_shapes)
  File "/Users/burg/git/background-removal-vue/node_modules/mil-web-dnn/src/graph_transpiler/webdnn/graph/converters/keras.py", line 573, in convert
    return converter.convert(input_shapes)
  File "/Users/burg/git/background-removal-vue/node_modules/mil-web-dnn/src/graph_transpiler/webdnn/graph/converters/keras.py", line 514, in convert
    assert inbound_node[3] == {}
AssertionError
```

When trying to debug the code it seems the problematic layer is:
`layer_class_name=Merge inbound_node=['conv2d_1', 0, 0, {'mask': [None, None]}]`

**Comment by Kiikurage** [comment]
Thank you for your report. In current implementation, not all features of Keras are supported and unsupported features are blocked by assertion. Reported assertion error is because of this restriction. Now we are implementing to support more features. please give us some time.

**Comment by milhidaka** [comment]
We just released v1.1.0.
This problem was solved in this release. If this problem still occurs, please reopen the issue and post the full model.
